.github/workflows/zulip_emoji_merge_delegate.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - bump/*
```

Comment:
Not sure if this be made more specific to:
```suggestion
      - bump/v*
```
And are there any other branches that we're frequently opening PRs to and merging with bors?

Reply:
I am happy either way: maybe let's try it with the more permissive `*` and restrict later, in case it gets confusing.

bors merge